src/gui/editors/AutomationEditor.cpp

```diff
@@ -215,7 +214,11 @@ void AutomationEditor::updateAfterClipChange()
 {
 	m_currentPosition = 0;
 
-	if( !validClip() )
+	setEnabled(validClip());
```

Comment:
setEnabled should happen in AutomationEditorWindow to disable the toolbar buttons